packages/acs-ui-common/src/logEvent.ts

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { AzureLogger } from '@azure/logger';
+
+/**
+ * @internal
+ */
+export type EventLevel = 'verbose' | 'info' | 'warning' | 'error';
```

Comment:
Can you import `AzureLogLevel` from `'@azure/logger'` instead of defining these ourselves